### PR TITLE
LIVE-2854: Make sure personality quizzes have the correct styling and work in dark mode

### PR DESCRIPTION
**Pull request by frankie297**
Some changes were requested to make personality quizzes work in dark mode for GLabs, this PR also makes sure the same is true for editorial. 

Based on these designs by @benwuersching: https://www.figma.com/file/00tb4ffXvbReLDmptMVrjx/Quiz-colours?node-id=1%3A679 

| GLabs Light Mode | GLabs Dark Mode |
| --- | --- |
|||

| Editorial Light Mode | Editorial Dark Mode |
| --- | --- |
|||

### CODE REVIEW

**Review by benwuersching — APPROVED** [review]
The colours look good, thank you Frankie.

File: ArticleTemplates/assets/scss/pillar/_colour.scss [hunk 1]
```diff
@@ -341,8 +341,76 @@
...
+                        border-top: 2px solid #707070;
+                    }
+                }
+            }
         }
     }
+
+    //for android
+    .body .dark-mode-on {
+        .element-atom {
```
Comment (marjisound) [inline ArticleTemplates/assets/scss/pillar/_colour.scss:391]: I think here you can create a mixin to share  the styles that are common between the ios and android. Something like 
```
@mixin apps-dark-mode-article {
        .element-atom {
            .quiz {
                &.personality-quiz {
                    .question__wrapper {
                        border-top: 2px solid #707070;
                    }

                    .question__text {
                        &::before {
                            color: #707070;
                        }
                    }

                    .quiz-scores,
                    .quiz-results {
                        border-color: #707070;
                        border-top: 2px solid #707070;
                    }
                }
            }
        }

.body .dark-mode-on {
  @include apps-dark-mode-article;
}

@media (prefers-color-scheme: dark) {
   @include apps-dark-mode-article;
}
}
```

**Review by marjisound — CHANGES_REQUESTED** [review]

File: ArticleTemplates/assets/scss/pillar/_colour.scss [hunk 2]
```diff
@@ -341,8 +341,76 @@
...
+                        border-color: #707070;
+                        border-top: 2px solid #707070;
+                    }
+                }
+            }
         }
     }
+
+    //for android
+    .body .dark-mode-on {
```
Comment (marjisound) [inline ArticleTemplates/assets/scss/pillar/_colour.scss:390]: I think here you can create a mixin to share the styles that are common between the ios and android. Something like
```
@mixin apps-dark-mode-article {
        .element-atom {
            .quiz {
                &.personality-quiz {
                    .question__wrapper {
                        border-top: 2px solid #707070;
                    }

                    .question__text {
                        &::before {
                            color: #707070;
                        }
                    }

                    .quiz-scores,
                    .quiz-results {
                        border-color: #707070;
                        border-top: 2px solid #707070;
                    }
                }
            }
        }

.body .dark-mode-on {
  @include apps-dark-mode-article;
}

@media (prefers-color-scheme: dark) {
   @include apps-dark-mode-article;
}
}
```

File: ArticleTemplates/assets/scss/pillar/_colour.scss [hunk 3]
```diff
@@ -341,8 +341,76 @@
...
+                .quiz-scores,
+                .quiz-results {
+                    border-color: color(brightness-7);
+                    border-top: 2px solid color(brightness-7);
+                }
+            }
+        }
+    }
+
+    @media (prefers-color-scheme: dark) {
```
Comment (marjisound) [inline ArticleTemplates/assets/scss/pillar/_colour.scss:365]: Do you know that we can use .ios & .android classes in here as well? to indicate if the style changes is for ios or android?

Comment (frankie297) [reply ArticleTemplates/assets/scss/pillar/_colour.scss:365]: the prefers-color-scheme is only for iOS and the dark-mode-on is only for android. adding the .ios and .android classes would have no benefit here

File: ArticleTemplates/assets/scss/pillar/_colour.scss (hunk 2 above)
Comment (frankie297) [reply ArticleTemplates/assets/scss/pillar/_colour.scss:390]: Good idea 👍

File: ArticleTemplates/assets/scss/pillar/_colour.scss (hunk 3 above)
Comment (marjisound) [reply ArticleTemplates/assets/scss/pillar/_colour.scss:365]: yeaa fair enough 👍

**Comment by paulmr** [comment]
For what its worth, I played around with this on my android phone just now and it all works perfectly. Nice work!

**Review by paulmr — APPROVED** [review]

**Review by marjisound — APPROVED** [review]
LGTM

**Pull request merged**